tsconfig.frontend.json

```diff
@@ -7,7 +7,7 @@
     /* Code generation */
     "inlineSources": true,
     "strictPropertyInitialization": false,
-    "lib": ["dom", "es5"]
+    "lib": ["dom", "es5", "dom.iterable"]
```

Comment:
Needed for `NodeList.entries()`/`NodeListOf<...>.entries()`, with this change I think some other code can be cleaned up too

Reply:
TIL why this is a separate lib, nice.